ha/core/system_health/model/entity_health.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import time
+import json
+
+from cortx.utils.conf_store import Conf
+from cortx.utils.log import Log
+from ha import const
+from ha.core.system_health import system_health_metadata
+from ha.core.error import HaEntityHealthException
+
+class EntityHealth:
+    """
+    Entity Health. This class implements a health object, which will be stored in store
+    for every component.
+    """
+
+    def __init__(self, event_timestamp, health_status, action_status,
```

Comment:
Could you please use type-hints everywhere? I think we have no reason to apply it partially.

Reply:
done.